_posts/2024-10-13-black-python-devs-ghana-event.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Black Python Devs Ghana Connect Meetup"
+date: 2024-10-13
+author: Abigail Afi Gbadago
+lang: en
+featured_image: /assets/images/bpd1.webp
+---
+
+On Saturday, August 17th, 2024, Stancy and Afi hosted the Black Python Devs Ghana Connect Meetup at Accra Mall, Pizza Inn. The event started at 1 PM with a warm introduction and welcome remarks from our moderator, Theresa Seyram Agbenyegah. We had an enthusiastic turnout of 22 participants, each of whom introduced themselves during an icebreaker session.
+
+Abigail Afi Gbadago then took the floor to share History: Community Formation Story, Members, the goal/mission successes so far(1st anniversary, GNOME donation link/QR scan) with participants. To get the conversation rolling, each participant was asked, **"Why Python?"** This led to a fascinating exchange, where everyone shared their journey and reasons for choosing Python as their preferred programming language.
+
+One of the highlights was an interesting story from an attendee about why he fell in love with Python. “ During my undergraduate studies at KNUST as an engineering student, I was tasked with creating sinusoidal plots and was free to use any programming language. Initially, I used MATLAB but found it somewhat challenging. After some research, I discovered Python, which was much easier to learn. In no time, I could generate beautiful plots using the matplotlib library. This marked the beginning of my journey with Python, eventually leading to proficiency in Django and the development of various applications. In 2020, I returned to Ghana at the peak of COVID-19. I found it challenging to find a job as a biomedical engineer, so I decided to pivot to tech in healthcare, leveraging my experience in writing and developing simple applications. Since the hospital didn't have a software engineer, delivering those services fell on me. I transitioned from developing command-line applications to creating and deploying full-stack applications. Although being the sole developer and handling all aspects of the software development lifecycle was tough, it allowed me to learn a lot, all on my own. Using Django and other Python packages made the process more manageable. Looking back, I realize that choosing Python was worth it ” ~ Douglas Amo-Sargon.
+
+![Black Python Devs Ghana Connect Meetup](/assets/images/bpd3.webp)
```

Comment:
```suggestion
![Black Python Devs Ghana Connect Meetup](/assets/images/bpd3.webp)

```

Reply:
Unless the linter complainsI would include a space to get things on separate lines.